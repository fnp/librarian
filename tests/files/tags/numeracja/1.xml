<strofa>
Jednych tam śmierć porwała kres życiu znacząca,/
Drugim z dala od ludzi Zeus życie rozkoszy/
Dał pełne i umieścił ich u ziemi końca,/
Gdzie żadna troska w sercu spokoju nie płoszy,/
W Krainie Szczęścia nad pienistym Oceanem,/
<numeracja start="100"/>Z dala od bogów; Kronos jest nad nimi panem,/
[Zwolniony z pęt przez bogów i ludzi rodzica./
Zeus im udzielił wielkiej czci, chwałą zaszczyca/
I wyniósł ich jak żadne w porównaniu z niemi/
Plemię z tych, które żyły na wszechpłodnej ziemi.]/
Cni bohaterzy! Trzykroć w roku owoc wszelki,/
Jak miód słodki, od ziemi biorą żywicielki.
</strofa>

<numeracja start="738"/>

<strofa>
Gdy masz przejść w bród przez piękne fale wiecznej rzeki,/
Módl się wprzód, umyj ręce i spójrz w prąd daleki./
Tym, co wchodzą do rzeki, nie mocząc rąk, prości,/
Bogowie odpłacają cierpieniem w przyszłości./
Gdy bogom uroczyste obiaty się palą,
</strofa>

<numeracja/>

<strofa>Pilnie strzeż się na bliźnich mieść potwarzy brzemię./
Mały zachód, trud lekki popuścić jej wodze,/
Lecz ciężko dźwigać, trudno ją zrzucić na ziemię./
Nigdy potwarz, jeżeli ją uprawia mnóstwo,/
Nie ginie całkiem; to też w swym rodzaju bóstwo.</strofa>
