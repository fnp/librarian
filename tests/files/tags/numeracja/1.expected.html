<div id="book-text">
<div class="stanza" id="i1">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Jednych tam śmierć porwała kres życiu znacząca,</div>
<div class="wl verse" id="f2">Drugim z dala od ludzi Zeus życie rozkoszy</div>
<div class="wl verse" id="f3">Dał pełne i umieścił ich u ziemi końca,</div>
<div class="wl verse" id="f4">Gdzie żadna troska w sercu spokoju nie płoszy,</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">W Krainie Szczęścia nad pienistym Oceanem,</div>
<a href="#f6" class="wl-num">100</a><div class="wl verse" id="f6">Z dala od bogów; Kronos jest nad nimi panem,</div>
<div class="wl verse" id="f7">[Zwolniony z pęt przez bogów i ludzi rodzica.</div>
<div class="wl verse" id="f8">Zeus im udzielił wielkiej czci, chwałą zaszczyca</div>
<div class="wl verse" id="f9">I wyniósł ich jak żadne w porównaniu z niemi</div>
<div class="wl verse" id="f10">Plemię z tych, które żyły na wszechpłodnej ziemi.]</div>
<a href="#f11" class="wl-num">105</a><div class="wl verse" id="f11">Cni bohaterzy! Trzykroć w roku owoc wszelki,</div>
<div class="wl verse" id="f12">Jak miód słodki, od ziemi biorą żywicielki.</div>
</div>
<div class="stanza" id="i2">
<div class="wl verse" id="f13">Gdy masz przejść w bród przez piękne fale wiecznej rzeki,</div>
<div class="wl verse" id="f14">Módl się wprzód, umyj ręce i spójrz w prąd daleki.</div>
<a href="#f15" class="wl-num">740</a><div class="wl verse" id="f15">Tym, co wchodzą do rzeki, nie mocząc rąk, prości,</div>
<div class="wl verse" id="f16">Bogowie odpłacają cierpieniem w przyszłości.</div>
<div class="wl verse" id="f17">Gdy bogom uroczyste obiaty się palą,</div>
</div>
<div class="stanza" id="i3">
<a href="#f18" class="wl-num">1</a><div class="wl verse" id="f18">Pilnie strzeż się na bliźnich mieść potwarzy brzemię.</div>
<div class="wl verse" id="f19">Mały zachód, trud lekki popuścić jej wodze,</div>
<div class="wl verse" id="f20">Lecz ciężko dźwigać, trudno ją zrzucić na ziemię.</div>
<div class="wl verse" id="f21">Nigdy potwarz, jeżeli ją uprawia mnóstwo,</div>
<a href="#f22" class="wl-num">5</a><div class="wl verse" id="f22">Nie ginie całkiem; to też w swym rodzaju bóstwo.</div>
</div>
</div>
